lib/elixir/lib/exception.ex

```diff
@@ -1257,6 +1278,27 @@ defmodule UndefinedFunctionError do
 end
 
 defmodule FunctionClauseError do
+  @moduledoc """
+  An exception raised when a function call doesn't match any defined clause.
+
+  The following fields of this exception are public and can be used freely:
```

Comment:
Editing this one too for consistency
```suggestion
  The following fields of this exception are public and can be accessed freely:
```